scripts/ci/send-email

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+import os
+from smtplib import SMTP
+from email.mime.text import MIMEText
+
+repo = os.environ['DRONE_REPO_NAME']
+if 'DRONE_TAG' in os.environ:
+    ref = os.environ['DRONE_TAG']
+else:
+    ref = os.environ['DRONE_BRANCH']
+bn = os.environ['DRONE_BUILD_NUMBER']
+
+msg = MIMEText(
+f'''See {os.environ['DRONE_BUILD_LINK']}
+
+-- 
+Hi Developer. Whaaat's happening? Listen. I'm gonna need you to fix
+your broken code. So if you could just go ahead and make sure it
+builds again, that'd be terrific. OK? Thanks!
+''')
+msg['From'] = 'Lumbergh CI <noreply-ci@pyr8.io>'
+msg['To'] = os.environ['EMAIL_RECIPIENTS']
+msg['Subject'] = f'''Build {bn} of {repo}:{ref} FAILED'''
+
+with SMTP(os.environ['EMAIL_HOST'], 587) as smtp:
+    smtp.starttls()
+    smtp.login(os.environ['EMAIL_USERNAME'], os.environ['EMAIL_PASSWORD'])
+    smtp.send_message(msg)
+    smtp.quit()
```

Comment:
Again, why not put this code into functions

Reply:
>  It's just a tiny little single-purpose loveless script. :-)
With several global variables and the burden to reorganize it into functions is shifted to the next person adding something the file.  It is a reasonable assumption to assume it will get changed eventually.

Reply:
I find it more readable than if there were functions. Feel free to checkout your changes and push into my branch.